macros/alter_table_properties.sql

```diff
@@ -730,6 +730,24 @@ ALTER VIEW archipelago_ethereum.fees  SET TBLPROPERTIES ('dune.public'='true',
                                                         'dune.data_explorer.contributors'='["0xRob"]');
 {% endset %}
 
+{% set ironbank_ethereum_itokens %}
```

Comment:
if you merge main from this repo back into your branch, this file will be deleted (as expected)
we will now have a 'post-hook' property in each model config block, like so:
https://github.com/duneanalytics/spellbook/blob/main/models/uniswap/ethereum/uniswap_v1_ethereum_trades.sql

where the four parameters are fed into this macro:
https://github.com/duneanalytics/spellbook/blob/main/macros/expose_spells.sql

note the macro doesn't need edited, just for reference to see how the code has been enhanced

Reply:
Got it. Fixed.